### PR TITLE
Fix generate errors

### PR DESCRIPTION
**Pull request by SwampDragons**
When running "make generate" against the whole codebase (not advised unless you want your computer to cry), the generator raises a fair number of "duplicate field" warnings -- This PR fixes the field collisions for HashiCorp maintained plugins.

### CODE REVIEW

File: builder/virtualbox/common/guest_additions_config.go [hunk 1]
```diff
@@ -52,14 +49,9 @@ type GuestAdditionsConfig struct {
 	GuestAdditionsURL string `mapstructure:"guest_additions_url" required:"false"`
 }
 
-func (c *GuestAdditionsConfig) Prepare(ctx *interpolate.Context) []error {
+func (c *GuestAdditionsConfig) Prepare(communicatorType string) []error {
```
Comment (azr) [inline builder/virtualbox/common/guest_additions_config.go:52]: Oh, good idea !!

**Review by azr — APPROVED** [review]
Super nice ! Yeah lots of duplicate fields in there 🙂 .

**Review by sylviamoss — APPROVED** [review]
Nice!

**Review by nywilken — APPROVED** [review]
LGTM

**Pull request merged**